client/api.c

```diff
@@ -258,11 +258,6 @@ TDNFGetSkipProblemOption(
 
     *pdwSkipProblem = SKIPPROBLEM_NONE;
 
-    if (strcasecmp(pTdnf->pArgs->ppszCmds[0], "check"))
```

Comment:
Why removing this section ?

Reply:
To enable `--skipconflicts` and `--skipobsoletes` for all commands, not just `check`. It may be useful in some cases.